src/query/plan/operator.cpp

```diff
@@ -321,6 +329,13 @@ bool CreateExpand::CreateExpandCursor::Pull(Frame &frame, ExecutionContext &cont
 
   if (!input_cursor_->Pull(frame, context)) return false;
 
+  if (context.auth_checker && !(context.auth_checker->Accept(*context.db_accessor, self_.edge_info_.edge_type,
+                                                             memgraph::auth::FineGrainedPermission::CREATE_DELETE) &&
+                                context.auth_checker->Accept(*context.db_accessor, self_.node_info_.labels,
+                                                             memgraph::auth::FineGrainedPermission::CREATE_DELETE))) {
+    spdlog::error("Error while creating an edge due to not having enough permission!");
```

Comment:
I would avoid logging this on any higher level than info or debug. Because it is done in every pull it can be easily produce thousands of log messages. Also, it is not an error from memgraphs perspective: it is the expected behavior that we prevent users from accessing vertices/edge to which they don't have access.

Reply:
This is also true for the rest of occurrences.